content/blog/gatsby-blog/index.md

```diff
@@ -0,0 +1,323 @@
+- **flexibility**: one of things that amazed us and that we are currently exploring (you will probably hear more about this in future blog posts) is that Gatsby is not only for static websites, it is pretty flexible and you can even create full fledged apps with it (without runtime server side rendering and some other considerations).
+- **development tools**: the whole development workflow is very nicely polished and the GraphiQL explorer of the entire site's data is a very nice quality of life improvement for developers.
+
+
+## Install Gatsby
+
+We used the [gatsby-starter-blog](https://www.gatsbyjs.org/starters/gatsbyjs/gatsby-starter-blog/) but take
+a look at other [community provided starters](https://www.gatsbyjs.org/starters/?v=2) for blogs or other types of pages
+
+```bash
+# Install Gatsby cli
+npm install -g gatsby-cli
+
+# Create a new gatsby site using the starter-blog starter
+gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog
+
+```
+
+Check the output to see how to run the app but at this point you have a working blog.
+
+These are the most relevant files:
+
+- `gatsby-config.js` to edit data about your site such as author, description, titles, etc
+- `gatsby-config.js` to add your Google Analytics id
+- `content/name-of-the-post/index.md` to see how to write posts in markdown
+- add `import "prismjs/themes/prism-tomorrow.css"` to `gatsby-browser.js` to enable **syntax highlighting** of code blocks. This starter uses [prism.js](https://prismjs.com/) but it does not add the css theme by default. Check prism.js documentation to see other themes available.
+
+
+And you are done! Told you! it is super fast and easy.
+
+
+
+Useful links
+- https://www.gatsbyjs.org/docs/quick-start/
+
+
+
+## Host your site in [Github Pages](https://pages.github.com/)
+
+Github pages is a static page hosting solution, if you are a software developer
+this hosting solution is pretty near to our daily roaming places so that is why I choose it.
+
+There are tons of alternatives but if you already host your page's code in github
+then this will be pretty easy to setup.
+
+Follow the instructions on [the website](https://pages.github.com/) to enable github pages.
+
+These are the extra steps I took for this blog
+
+### 1 make `develop` your default branch (only for _project site_)
+
+Since this blog qualifies as a *project site* (per repo) instead of a *user or organization site*
+we need to have the following repo branch configuration:
+
+- `develop`: where our gatsby code lives
+- `master`: where our gatsby-generated static files for our website are stored.
+
+These generated static files will be the ones that github pages will deploy and show.
+
+Have in mind that you will be managing two code bases, one is your gatsby site, the code that you will
```

Comment:
I think a more appropriate expression would be "keep in mind" since "have in mind" expresses a different thought: https://idioms.thefreedictionary.com/have+in+mind